backend/internal/consumer/storage_updater.go

```diff
@@ -227,11 +233,11 @@ func (u *storageUpdater) handleProcessInstance(untypedRecord *UntypedRecord) err
 		timestamp,
 	)
 	if err != nil {
-		log.Printf(
+		zap.L().Error(
 			"Failed to log event to audit log: position %d, element id %s, err: %v",
```

Comment:
Change this to `Failed to log event to audit log` without `: position %d, element id %s, err: %v`